localgov.profile

```diff
@@ -45,3 +45,12 @@ function localgov_post_install_task(): void {
     $hook();
   });
 }
+
+/**
+ * Enable masquerade module.
+ */
+function localgov_update_10001() {
+  if (!\Drupal::service('module_handler')->moduleExists('masquerade')) {
+    \Drupal::service('module_installer')->install(['masquerade']);
```

Comment:
Hi @markconroy thanks for this.

I'm not sure we want to enable masquerade for all existing sites. 

If I'm using https://www.drupal.org/project/msqrole (which we are on a number of sites) then I don't want this as well. 

I suggest we remove the update hook and let people know in the release notes that it is now available if desired.

Do we also want to add permissions for this to the default install?